### PR TITLE
Mk  term edit animation iteration 3410

### LINKED ISSUE #3410: [Edit] CSS [Edit] CSS Animations: animation-iteration-count
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like an edit a term within the CSS Animations entry. The entry lives in docs/content/css/concepts/animations/terms/animation-iteration-count/animation-iteration-count.md

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add supplemental media (preferred .png or .gif, <1MB).
2) Media (images, gifs) should accurately represent the existing code blocks in the example sections and fit in-line.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by marjoriekohn**
### Description

Topic: CSS Concepts - Animations
Edited File: animation-iteration-count.md
Created Files:
- animation-iteration-count-1.gif
- animation-iteration-count-2.gif

Closes #3410 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
I did a first review and left a couple of comments.

File: content/css/concepts/animations/terms/animation-iteration-count/animation-iteration-count.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ div {
 }
 ```
 
+![animation-iteration-count-example-2](../../../../../../media/css-animation-iteration-count-fade.gif)
```
[the remark this answers is not in the document]
Comment (marjoriekohn) [reply content/css/concepts/animations/terms/animation-iteration-count/animation-iteration-count.md]: done

**Review by SSwiniarski — APPROVED** [review]
@marjoriekohn Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@marjoriekohn @SSwiniarski Looks good

**Pull request merged**